apps/dashboard/src/lib/components/Org/sidebarIcons.svelte

```diff
@@ -0,0 +1,37 @@
+<script>
+  import HomeIcon from '../Icons/HomeIcon.svelte';
+  import SiteSettingsIcon from '../Icons/SiteSettingsIcon.svelte';
+  import QuizIcon from '../Icons/QuizIcon.svelte';
+  import AudienceIcon from '../Icons/AudienceIcon.svelte';
+  import ForumIcon from 'carbon-icons-svelte/lib/Forum.svelte';
+  import { SettingsAdjust } from 'carbon-icons-svelte';
+  import CourseIcon from '../Icons/CourseIcon.svelte';
+
+  export let iconId;
+
+  let ICON_CLASS = {
+    dashboard: 'dashboard',
+    courses: 'courses',
+    community: 'community',
+    site: 'site',
+    quiz: 'quiz',
+    audience: 'audience',
+    setup: 'setup'
+  };
```

Comment:
1. Make `ICON_CLASS` a const
2. Please verify that there is no type errors in this file. `iconId` should have a type